iresize.lisp

```diff
@@ -71,24 +72,25 @@
         (focus-window (current-window))))))
 
 (deficommand (iresize tile-group)
-    (((kbd "Up") "resize-direction up")
-     ((kbd "C-p") "resize-direction up")
-     ((kbd "p") "resize-direction up")
-     ((kbd "k") "resize-direction up")
-
-     ((kbd "Down") "resize-direction down")
-     ((kbd "C-n") "resize-direction down")
-     ((kbd "n") "resize-direction down")
-     ((kbd "j") "resize-direction down")
-
-     ((kbd "Left") "resize-direction left")
-     ((kbd "C-b") "resize-direction left")
-     ((kbd "b") "resize-direction left")
-     ((kbd "h") "resize-direction left")
-
-     ((kbd "Right") "resize-direction right")
-     ((kbd "C-f") "resize-direction right")
-     ((kbd "f") "resize-direction right")
-     ((kbd "l") "resize-direction right"))
-  :pre-start-fn #'setup-iresize
-  :cleanup-fn #'resize-unhide)
+
```

Comment:
Below I've pasted a more idiomatic indentation style (If you are using Emacs, just C-M-Space to select the whole form and then M-\ to indent)
```lisp
(deficommand (iresize tile-group) (:bootstrap-fn #'setup-iresize
                                   :cleanup-fn #'resize-unhide)
  ((kbd "Up") "resize-direction up")
  ((kbd "Up") "resize-direction up")
  ((kbd "C-p") "resize-direction up")
  ((kbd "p") "resize-direction up")
  ((kbd "k") "resize-direction up")

  ((kbd "Down") "resize-direction down")
  ((kbd "C-n") "resize-direction down")
  ((kbd "n") "resize-direction down")
  ((kbd "j") "resize-direction down")

  ((kbd "Left") "resize-direction left")
  ((kbd "C-b") "resize-direction left")
  ((kbd "b") "resize-direction left")
  ((kbd "h") "resize-direction left")

  ((kbd "Right") "resize-direction right")
  ((kbd "C-f") "resize-direction right")
  ((kbd "f") "resize-direction right")
  ((kbd "l") "resize-direction right"))
```